JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -15,479 +15,456 @@
+  
+  if (!is.null(jaspResults[["scaleTable"]])) return()
+  
+  # Create table
+  scaleTable <- createJaspTable(title = "Scale Reliability Statistics")
+  dependList <- c("variables", "confAlpha", "mcDonaldScale", "alphaScale", "meanScale",
+                  "alphaScaleStandardized", "gutmannScale", "glbScale", "confAlpha",
+                  "averageInterItemCor",  "sdScale", "missingValues", "confAlphaLevel")
+  scaleTable$dependOn(dependList)
+  scaleTable$showSpecifiedColumnsOnly <- TRUE
+  scaleTable$position <- 1
+  
+  # Add columns to table
+  scaleTable$addColumnInfo(name = "case", title = "", type = "string")
+  if (options$meanScale)
+    scaleTable$addColumnInfo(name = "mu", title = "mean", type = "number")
+  if (options$sdScale)
+    scaleTable$addColumnInfo(name = "sd", title = "sd", type = "number")
+  if (options$mcDonaldScale)
+    scaleTable$addColumnInfo(name = "omega", title = "McDonald's \u03C9",     
+                             type = "number")
+  if (options$alphaScale)
+    scaleTable$addColumnInfo(name = "alpha", title = "Cronbach's \u03B1",
+                             type = "number")
+  if (options$gutmannScale)
+    scaleTable$addColumnInfo(name = "lambda", title = "Gutmann's \u03BB6",
+                             type = "number")
+  if (options$glbScale)
+    scaleTable$addColumnInfo(name = "glb", title = "Greatest lower bound",
+                             type = "number")
+  if (options$averageInterItemCor)
+    scaleTable$addColumnInfo(name = "rho", title = "Average interitem correlation",
+                             type = "number")
+  if (options$confAlpha){
+    overTitle <- paste0(100*options$confAlphaLevel, "% Confidence Interval")
+    scaleTable$addColumnInfo(name = "lower", title = "Lower", type = "number", 
+                             overtitle = overTitle)
+    scaleTable$addColumnInfo(name = "upper", title = "Upper", type = "number", 
+                             overtitle = overTitle)
+  }
+  
+  if (options$missingValues == "excludeCasesListwise") 
+    exclwise = " listwise"
+  else 
+    exclwise = " pairwise"
+  
+  variables <- unlist(options$variables)
+  nObs      <- nrow(dataset)
+  nExcluded <- sum(!complete.cases(dataset))
+  nValid    <- nObs - nExcluded
+  
+  if(length(variables) >= 2) {
+    message <- sprintf("Of the observations, %d were used, %d were excluded%s, 
+                     and %d were provided.", nValid, nExcluded, exclwise, nObs)
+    if (options$glbScale && length(variables) == 2)
+      message <- paste(message, "Warning: Greatest lower bound can 
+    only be calculated for three or more variables.")
+    scaleTable$addFootnote(message, symbol = "<em>Note.</em>")
+  }
+  
+  jaspResults[["scaleTable"]] <- scaleTable
+  
+  # Compute/get the results
+  relyFit <- .reliabilityComputeResults(jaspResults, dataset, options, ready)
+  
+  res <- try(.reliabilityScaleFill(jaspResults, dataset, options, relyFit, ready))
```

Comment:
I don't think `reliabilityScaleFill` can actually fail, so this `try` bit is not necessary